src/app/campaign/store/effects/campaign.effects.ts

```diff
@@ -259,8 +259,15 @@ export class CampaignEffects {
             campaign,
             flights: flights.map(flight => ({
               ...flight,
```

Comment:
Two for one! How convenient! Clearing the flight dates is a business rule that got left behind when we added contract dates and also did the fudged date fields, and it was those Moments causing the error. If we want the dates, we can't pass them through this router state as Moments, they would need to be strings or something.